File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/struct/p22.c
/* passing structure to a function*/

#include<stdio.h>
struct Demo{

	int x;
	int y;
};
void main(){
	struct Demo obj={10,20.20};
	int arr[]={10,20,30,40,50};

	printf("%p\n",&arr[0]);
	printf("%p\n",arr);
	printf("%p\n",&obj.x);
	printf("%p\n",obj);
	printf("%p\n",&obj);
}

File: java-main/java-main/Array/p22.java
class Demo{
	void fun(int xarr[]){
		for(int x : xarr){
		System.out.println(x);
		}
		xarr[0]=50;
	}

	public static void main(String[]args){
		Demo obj = new Demo();
		int arr[]={10,20,30,40,50};
		for(int x : arr){
			System.out.println(x);
		}
		for(int x : arr){
			System.out.println(x);
			
		}
	}
}

